docs/source/samples/depth_preview.rst

```diff
@@ -5,11 +5,16 @@ This example shows how to set the SGBM (semi-global-matching) disparity-depth no
 over XLink to transfer the results to the host real-time, and displays the depth map in OpenCV.
 Note that disparity is used in this case, as it colorizes in a more intuitive way.
 Below is also a preview of using different median filters side-by-side on a depth image.
+There are 3 depth modes which you can select inside the code:
+#. lr_check: computes disparity with sub-pixel interpolation (5 fractional bits), suitable for long range
+#. extended_disparity: mirror rectified frames: true to have disparity/depth normal (non-mirrored)
```

Comment:
Cross link extended_disparity description to the following link:
https://docs.luxonis.com/en/latest/pages/faq/#extended-disparity-depth-mode
LR-check to : https://docs.luxonis.com/en/latest/pages/faq/#left-right-check-depth-mode
Subpixel: https://docs.luxonis.com/en/latest/pages/faq/#subpixel-disparity-depth-mode

Same for other examples.

Reply:
Update:
Use the links from stereo node description: https://docs.luxonis.com/projects/api/en/latest/components/nodes/stereo_depth/#left-right-check
Since the ones from FAQ are redundant.
@csaba-luxonis